### PR TITLE
changelog: pgvector support

### PR DESCRIPTION
**Pull request by davlgd**
## Describe your PR

Add a changelog entry for pgvector support, linked to #200 

## Checklist

- [ ] My PR is related to an opened issue : #
- [ ] I've read the [contributing guidelines](../CONTRIBUTING.md)

## Reviewes
_Who should review these changes?_ @

### CODE REVIEW

**Review by aurrelhebert — APPROVED** [review]
LGTM! :rocket:

**Review by aurrelhebert — APPROVED** [review]

File: content/changelog/2024-02-22-postgresql-pgvector-support.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    link: https://github.com/aurrelhebert
+    image: https://github.com/aurrelhebert.png?size=40
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+description: Your wishes are our commands
+excludeSearch: true
+---
+
+PostgreSQL databases managed by Clever Cloud come with [lots of extensions activated by default](/doc/addons/postgresql/#default-extensions). There are also [some you can ask for](/doc/addons/postgresql/#on-demand-extensions), and we'll set them up, like `pg_cron`, `pgtap` or `timescaledb`.
```
Comment (draescherl) [inline content/changelog/2024-02-22-postgresql-pgvector-support.md:21]:
```suggestion
PostgreSQL databases managed by Clever Cloud come with [lots of extensions enabled by default](/doc/addons/postgresql/#default-extensions). There are also [some you can ask for](/doc/addons/postgresql/#on-demand-extensions), and we'll set them up, like `pg_cron`, `pgtap` or `timescaledb`.
```

File: content/changelog/2024-02-22-postgresql-pgvector-support.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Recently, some customers also asked us a `pgvector` support, needed for specific (AI) workloads. So we started to work on it and integrated this extension to our workflow. We are now able to provide it on-demand.
```
Comment (draescherl) [inline content/changelog/2024-02-22-postgresql-pgvector-support.md:23]:
```suggestion
We recently noticed an increased interest from our clients in supporting the `pgvector` extension, needed for specific (AI) workloads. In order to accommodate these needs, we have packaged the extension for our systems and are now able to provide it on-demand.
```

File: content/changelog/2024-02-22-postgresql-pgvector-support.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+So if you need `pgvector`, just raise a ticket to our support team through [the Console Ticket Center](https://console.clever-cloud.com/ticket-center-choice). And if there is any other extension you need, feel free to ask, we will check how easy it is for us to provide it.
```
Comment (draescherl) [inline content/changelog/2024-02-22-postgresql-pgvector-support.md:25]:
```suggestion
If you need `pgvector`, open a ticket to our support team through [the Console Ticket Center](https://console.clever-cloud.com/ticket-center-choice). If there are any other extension you need, feel free to ask, we will try our best to adapt to your use-case.
```

**Review by draescherl — CHANGES_REQUESTED** [review]

**Review by cnivolle — APPROVED** [review]

**Comment by juliamrch** [comment]
So no one in this PR has even checked the flags and proposed suggestions ? https://github.com/CleverCloud/documentation/pull/202/files

**Pull request merged**